docs/codeql/codeql-for-visual-studio-code/testing-codeql-queries-in-visual-studio-code.rst

```diff
@@ -12,27 +12,31 @@ About testing queries in VS Code
 
 To ensure that your CodeQL queries produce the expected results, you can run tests that compare the expected query results with the actual results.
 
-The CodeQL extension automatically prompts VS Code to install the Test Explorer extension as a dependency. The Test Explorer displays any workspace folders with a name ending in ``-tests`` and provides a UI for exploring and running tests in those folders.
+The CodeQL extension automatically registers itself with the **Test Explorer** view. The **Test Explorer** view displays all tests found in your current workspace and provides a UI for exploring and running tests in your workspace.
 
 For more information about how CodeQL tests work, see "`Testing custom queries <https://docs.github.com/en/code-security/codeql-cli/using-the-codeql-cli/testing-custom-queries>`__" in the CLI help.
 
 Testing the results of your queries
 -----------------------------------
 
-1. Open the Test Explorer view in the sidebar.
+1. Open the **Test Explorer** view in the sidebar.
 
    .. image:: ../images/codeql-for-visual-studio-code/open-test-explorer.png
       :width: 350
       :alt: Open the Test Explorer view
 
 2. To run a specific test, hover over the file or folder name and click the play button. To run all tests in your workspace, click the play button at the top of the view. If a test takes too long to run, you can click the stop button at the top of the view to cancel the test.
-3. The icons show whether a test passed or failed. If it failed, right-click the failed test and click **CodeQL: Show Test Output Differences** to display the differences between the expected output and the actual output.
```

Comment:
Side note: I see that we still have the "Show Test Output Differences" command in the [extension code](https://github.com/github/vscode-codeql/blob/2ba23ceead55208e616587be12696bcc9a28be66/extensions/ql-vscode/src/query-testing/test-manager.ts#L217), but it doesn't seem to be used anywhere 😅 I wonder if we can delete it 👀 🤔

Reply:
I'm not sure if this code is reachable, but it seems like we should remove it if it's not.